host/ProtocolGateway.Host.Common/Bootstrapper.cs

```diff
@@ -45,7 +46,7 @@ public class Bootstrapper
         readonly IotHubClientSettings iotHubClientSettings;
         readonly IMessageAddressConverter topicNameConverter;
 
-        public Bootstrapper(ISettingsProvider settingsProvider, ISessionStatePersistenceProvider sessionStateManager, IQos2StatePersistenceProvider qos2StateProvider)
+        public Bootstrapper(ISettingsProvider settingsProvider, ISessionStatePersistenceProvider sessionStateManager, IQos2StatePersistenceProvider qos2StateProvider, List<string> inboundTemplates = null, List<string> outboundTemplates = null)
```

Comment:
Done. Created a private constructor to keep them using same logic and created two public constructors to expose variant